lib/index.js

```diff
@@ -44,6 +44,9 @@ const createStream = options => {
   return stream;
 };
 
+// Download the file in chunks, in this case 10MB, anything
+// over this will cause youtube to throttle the download
+const dlChunkSize = 10485760;
```

Comment:
Will it be better to add the ability to change the `DL_CHUNK_SIZE` through the options?

Reply:
maybe it's worth it. for tests, having it configurable will allow you to test with a smaller file, it would be nice not to have to commit a file that's over 10mb to the repo just for a test.

it could also be nice for users. frequently, users of the library will tweak the options little by little and finding out what works best for them. this has happened most recently with `highWaterMark`. we should provide sensible defaults, but also configurabilty, specially for an ever-changing unofficial public api.